### PR TITLE
Feature/taoe cf enhancements

### PR DESCRIPTION
**Pull request by wheat4714**
These changes improve the TAoE CF similar to the QxR CF changes. This smoothes out the frictions of grabs and imports.

### CODE REVIEW

File: imports/custom_formats/sonarr/custom formats (sonarr - master).json [hunk 1]
```diff
@@ -12024,14 +12024,56 @@
...
+                "negate": true,
+                "required": true,
                 "fields": [
                     {
                         "order": 0,
                         "name": "value",
                         "label": "Regular Expression",
                         "helpText": "Custom Format RegEx is Case Insensitive",
-                        "value": "TAoE",
+                        "value": "H.265",
```
Comment (santiagosayshey) [inline imports/custom_formats/sonarr/custom formats (sonarr - master).json:12076]: This needs to be changed

**Review by santiagosayshey — CHANGES_REQUESTED** [review]

**Comment by santiagosayshey** [comment]
@wheat4714 Just need to fix up the edge cases where a release group comes after a delimeter without taoe.

**Comment by santiagosayshey** [comment]
note to myself later
- need to add radarr equiv

**Comment by wheat4714** [comment]
https://regex101.com/r/NZ0fOE/9

**Review by wheat4714 — COMMENTED** [review]
I made the changes

**Review by santiagosayshey — APPROVED** [review]
Approved!

**Pull request merged**